packages/cozy-client/examples/delete-files.js

```diff
@@ -0,0 +1,43 @@
+const { createClientInteractive } = require('cozy-client')
+const {
+  schema,
+  createFolderWithFilesBulk,
+  deleteAllFilesWithoutPath,
+  getAllFiles,
+  deleteFile
+} = require('./helpers/file')
+const { ArgumentParser } = require('argparse')
+
+global.fetch = require('node-fetch') // in the browser we have native fetch
+
+/**
+ * Several utilities for files management:
+ * - Create a folder with files
+ * - Delete it
+ * - Delete all the files without path that are not trashed
+ * - Get all the files
+ *
+ */
+const main = async () => {
+  const parser = new ArgumentParser()
+  parser.addArgument('--url', { defaultValue: 'http://cozy.localhost:8080' })
+  const args = parser.parseArgs()
+
+  const client = await createClientInteractive({
+    scope: ['io.cozy.files'],
+    uri: args.url,
+    schema,
+    oauth: {
+      softwareID: 'io.cozy.client.cli'
+    }
+  })
+
+  const count = 10
+
+  const folder = await createFolderWithFilesBulk(client, count)
+  await deleteFile(client, folder)
+  await deleteAllFilesWithoutPath(client, true)
+  await getAllFiles(client)
```

Comment:
Why do we get all the files without reading the response?

Reply:
ok removed